personas.md

```diff
@@ -0,0 +1,126 @@
+# Personas & Pathways
+
+Personas are character sketches of fictional contributors and users who might do something with the Turing Way.
+Following the [Mozilla Open Leadership training](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/bring-on-contributors-using-personas-and-pathways/), we created personas for contributors to the Turing Way based on participants in past trainings and events, and have created personas for users based on our target audiences.
+This will help us in our development of the Turing Way community such as through the Readme and Contributing Guidelines.
```

Comment:
```suggestion
This will help us in our development of the Turing Way community such as through the [README](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md) and [Contributing Guidelines](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md).
```